units/systemd-machine-id-commit.service

```diff
@@ -12,8 +12,9 @@ Description=Commit a transient machine-id on disk
 Documentation=man:systemd-machine-id-commit.service(8)
 DefaultDependencies=no
 Conflicts=shutdown.target
-Before=sysinit.target shutdown.target
-After=local-fs.target
+Before=shutdown.target
+After=local-fs.target first-boot-complete.target
+Wants=first-boot-complete.target
```

Comment:
no, first-boot-complete.target should be "passive" i think, as mentioned, i.e. pulled in by the stuff ordered before it, not the stuff ordered after it. i.e. please drop the Wants= but keep the After=